src/types/any.rs

```diff
@@ -1112,8 +1114,10 @@ pub trait PyAnyMethods<'py> {
     ///
     /// # fn main() -> PyResult<()> {
     /// Python::with_gil(|py| -> PyResult<()> {
-    ///     let a: &PyInt = 0_u8.into_py(py).into_ref(py).downcast()?;
-    ///     let b: &PyInt = 42_u8.into_py(py).into_ref(py).downcast()?;
+    ///     let a = 0_u8.into_py(py).into_bound(py);
+    ///     let b = 42_u8.into_py(py).into_bound(py);
+    ///     let a: &Bound<'_, PyInt> = a.downcast()?;
+    ///     let b: &Bound<'_, PyInt> = b.downcast()?;
```

Comment:
Similar suggestion to use `.downcast_into()` to keep these expressions one-liners.